### PR TITLE
bright/darkmode cookie fix

### LINKED ISSUE #1534: Darkmode setting is not remembered as cookie

### PR DESCRIPTION
**Pull request by ovbm**
Fixes #1534 

brightModeEnabled is enabled and application state now reads the cookie and sets state accordingly, I tried running this locally in chrome but it didn't open in darkmode after closing tab and reopening... any ideas?

### CODE REVIEW

File: web/src/reducers/index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const initialApplicationState = {
   callerLocation: null,
   clientType: window.isCordova ? 'mobileapp' : 'web',
   colorBlindModeEnabled: Cookies.get('colorBlindModeEnabled') === 'true' || false,
-  brightModeEnabled: Cookies.get('brightModeEnabled') === 'true' || true,
+  brightModeEnabled: Cookies.get('brightModeEnabled') === 'true' || false,
```
Comment (corradio) [inline web/src/reducers/index.js]: you're defaulting to false here. I don't think we want that?

**Comment by corradio** [comment]
hmm so the state has brightModeEnabled = false?

**Comment by ovbm** [comment]
Ah, i misinterpreted the code.

**Comment by corradio** [comment]
@ovbm I've fixed the cookies. However, now, when starting in brightmode, some text is still black instead of white:

image

File: web/src/main.js [hunk 2]
```diff
@@ -1383,11 +1370,17 @@ observe(state => state.application.legendVisible, (legendVisible) => {
 // Observe for left panel collapse
 observe(state => state.application.isLeftPanelCollapsed, (_, state) =>
   renderLeftPanelCollapseButton(state));
-
 // Observe for search query change
 observe(state => state.application.searchQuery, (searchQuery, state) => {
   zoneList.filterZonesByQuery(searchQuery);
 });
+// Observe for brightmode change
+observe(state => state.application.brightModeEnabled, (brightModeEnabled, _) => {
```
Comment (corradio) [inline web/src/main.js]: @ovbm This is the key change: I had to observe for a redux state change (before, we dispatched to redux but didn't listen to state changes)

**Pull request merged**